packages/react-storage/src/components/StorageBrowser/displayText/libraries/en/uploadView.ts

```diff
@@ -2,6 +2,17 @@ import { DEFAULT_ACTION_VIEW_DISPLAY_TEXT } from './shared';
 import { DefaultUploadViewDisplayText } from '../../types';
 import { isFileTooBig } from '../../../validators';
 
+function getMessage(
+  count: number,
+  total: number | undefined,
+  result?: string
+): string {
+  const quantity = count === total ? 'All' : String(count);
+  const pluralize = count > 1 || count === total ? 'files' : 'file';
+
+  return `${quantity} ${pluralize}${result ? ` ${result}` : ''}`;
+}
```

Comment:
The initial intention of not using a util in the message resolving was to avoid inflexibility as util retuns and natural language generally do not align easily. Preference is to lean towards verbose code for improvements the messages we provide to end users

Reply:
simplified it a bit. i'm not adding new logic. it's just moving repeated logic from the existing function